PublicMapping/districtbuilder
# Add histogram and text description to evaluate mode Competitiveness page

## Overview

- Adds the `visx` visualization library to support rendering of charts
- Adds a histogram grouping a count of districts for each of the 5 political leanings buckets
- Adds the `shortText` and `longText` specified in #843 to the Competitiveness metric page

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/124822659-2d8b4380-df3e-11eb-9802-d1c6bdac2de3.png)


### Notes

- I ran into some type ugliness around the visx scales, and it also seems like my current computation of the PVI buckets could likely be improved a little bit. I am curious if you have any better suggestions for the `computeRowBucket` function - it seems like we would probably want to refactor this into the existing `computeRowFill` function if there is not a better solution.

## Testing Instructions

- `./scripts/update`
- Open a completed project for a state with political data (e.g., IL)
- Go to Evaluate - Competitiveness, expect histogram to be displayed
- Select a different year and expect data in histogram to change

Closes #837 and #843
